env0/resource_workflow_trigger.go

```diff
@@ -0,0 +1,90 @@
+package env0
+
+import (
+	"context"
+	"log"
+
+	"github.com/env0/terraform-provider-env0/client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceWorkflowTrigger() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceWorkflowTriggerCreate,
+		ReadContext:   resourceWorkflowTriggerRead,
+		DeleteContext: resourceWorkflowTriggerDelete,
+		Description:   "cannot be used with env0_workflow_triggers",
+
+		Schema: map[string]*schema.Schema{
+			"environment_id": {
+				Type:        schema.TypeString,
+				Description: "id of the source environment",
+				Required:    true,
+				ForceNew:    true,
+			},
+			"downstream_environment_id": {
+				Type:        schema.TypeString,
+				Description: "environment to trigger",
+				Required:    true,
+				ForceNew:    true,
+			},
+		},
+	}
+}
+
+func resourceWorkflowTriggerRead(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	apiClient := meta.(client.ApiClientInterface)
+
+	environmentId := d.Get("environment_id").(string)
+	downstreamEnvironmentId := d.Get("downstream_environment_id").(string)
+
+	triggers, err := apiClient.WorkflowTrigger(environmentId)
+
+	if err != nil {
+		return diag.Errorf("could not get workflow triggers: %v", err)
```

Comment:
is it not possible that when there are no workflow triggers we are drifted? 🤔

Reply:
so if someone deleted the trigger we would not recognize it as drift but add it? 🤔